service/src/registry.ts

```diff
@@ -444,6 +491,51 @@ export class Registry {
     }, done);
   }
 
+  /**
+   * @method              module:azure-iothub.Registry#exportDevicesToBlob
+   * @description         Export devices to a blob in a bulk job.
+   * @param {String}      outputBlobContainerUri  The URI to a container where a blob will be created with logs of the export process.
+   * @param {Boolean}     excludeKeys             Boolean indicating whether security keys should be excluded from the exported data.
+   * @param {Function}    [done]                  The optional function to call when the job has been created, with two arguments: an error object if an
+   *                                              an error happened, (null otherwise) and the job status that can be used to track progress of the devices export.
+   * @returns {Promise<Registry.JobStatus> | void} Promise if no callback function was passed, void otherwise.
+   */
+  exportDevicesToBlobByIdentity(outputBlobContainerUri: string, excludeKeys: boolean, done: Callback<Registry.JobStatus>): void;
+  exportDevicesToBlobByIdentity(outputBlobContainerUri: string, excludeKeys: boolean): Promise<Registry.JobStatus>;
+  exportDevicesToBlobByIdentity(outputBlobContainerUri: string, excludeKeys: boolean, done?: Callback<Registry.JobStatus>): Promise<Registry.JobStatus> | void {
+    return callbackToPromise((_callback) => {
+      /* Codes_SRS_NODE_IOTHUB_REGISTRY_16_004: [A ReferenceError shall be thrown if outputBlobContainerUri is falsy] */
```

Comment:
>Y_16_00 [](start = 38, length = 7)

I don't think actually created new specs for these functions and added them in the devdoc.

Reply:
I will update this

---
In reply to: [388029843](https://github.com/Azure/azure-iot-sdk-node/pull/709#discussion_r388029843) [](ancestors = 388029843)